### ISSUE TITLE
unmet dependency `jsdom/lib/old-api`

### ISSUE THREAD

**Issue by ktravers** [learn-co-curriculum/moving-things-with-javascript#17]
getting the following error when i run `learn test`:

```
1) index "before all" hook:
     Error: Cannot find module 'jsdom/lib/old-api'
      at require (internal/module.js:11:18)
      at Context.<anonymous> (node_modules/mocha-jsdom/index.js:53:5)
```

**Comment by lisaychuang** [comment]
In `package.json` change this: `"mocha-jsdom": "^1.1.0",` to `"mocha-jsdom": "2.0.0",` 
and run `npm install`

If that doesn't work, 
change: `"mocha-jsdom": "^1.1.0",` to `"mocha-jsdom": "~1.1.0",` and run `npm install`

**Comment by musthavepockets** [comment]
Thank you! The first suggestion worked for me after saving & quitting Learn IDE and then reopening the lesson.

**Comment by ktravers** [comment]
should be fixed by #18

**Comment by drakeltheryuujin** [comment]
Thank you for spotting this issue and providing feedback.

We have updated the materials and believe your issue to have been resolved.

We apologize for any frustration you might have encountered during this process but thank you for helping us ensure that those who follow in your path will not encounter the same problems that you faced.

If you do not believe that this issue has been addressed, please re-open this issue. 💙
